packages/core/src/textures/GLTexture.ts

```diff
@@ -70,5 +71,10 @@ export class GLTexture
          * @member {number}
          */
         this.internalFormat = 5121;
+
+        /**
+         * @member {number}
+         */
```

Comment:
Minor comment, this is not necessary, if you add a documentation block above `public samplerType: number` see #7159

Reply:
done, can you check please?